rules/windows/defense_evasion_via_filter_manager.toml

```
@@ -17,14 +42,62 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Potential Evasion via Filter Manager"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Potential Evasion via Filter Manager
+
+A file system filter driver, or minifilter, is a specialized type of filter driver designed to intercept and modify I/O requests sent to a file system or another filter driver. Minifilters are used by a wide range of security software, including EDR, antivirus, backup agents, encryption products, etc.
+
+Having such importance in security use cases, attackers may try to unload minifilters to avoid protections such as malware detection, file system monitoring, and behavior-based detections.
```

I removed the first part of the sentence because I think you already thoroughly explained their role in security use cases. 

```suggestion
Attackers may try to unload minifilters to avoid protections such as malware detection, file system monitoring, and behavior-based detections.
```